scripts/create-release-branch.js

```diff
@@ -68,9 +68,6 @@ if (isRemoteReleaseBranchExist) {
   process.exit(-1);
 }
 
-// Create the new release branch and switch to it
-shell.exec(`git checkout -b ${releaseBranchName}`);
-
```

Comment:
So I'm lacking a bit in the how releases work department, but am I right that this script is assuming we're on the main branch to start?  So previously we were committing the changes to the release branch, and now we're committing directly to the main branch?  Do we ever push the changes back to the remote main branch?   I'm debating how I feel about committing directly to main.  Typically you don't want commits to be directly to the main branch.

Reply:
actually never mind I finally processed the code above...so we're on develop.  The question is now how do we get from the release branch back to main? Usually you only want to catch up develop with main (otherwise why does main even exist?).  If I'm way off here please just ping me and straighten me out.